MMM-Spotify.js

```diff
@@ -13,9 +13,11 @@ Module.register("MMM-Spotify", {
     iconify: "https://code.iconify.design/1/1.0.6/iconify.min.js",
```

Comment:
can we add "minimalistBar" to the list of styles available?

Reply:
no, it will be set automaticaly ;) (depend of module position)

because on other position it's buggy